lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -443,7 +471,11 @@ export abstract class ClientApplication {
             correlationId: correlationId
         };
         this.preflightBrowserEnvironmentCheck(InteractionType.Silent);
-        const ssoSilentMeasurement = this.performanceClient.startMeasurement(PerformanceEvents.SsoSilent, correlationId);
+        this.ssoSilentMeasurement = this.performanceClient.startMeasurement(PerformanceEvents.SsoSilent, correlationId);
+        this.ssoSilentMeasurement?.addStaticFields({
+            visibilityChange:false
```

Comment:
Don't need to add this at all if it doesn't happen. In any case I don't think our pattern should involve overwriting static fields after they've already been assigned

Reply:
Looks like it's fixed already.